tools/bin/ldflags

```diff
@@ -3,6 +3,7 @@
 cd "$(dirname "$0")"
 
 COMMIT_SHA=${COMMIT_SHA:-$(git rev-parse HEAD)}
+VERSION_TAG=${VERSION_TAG:-$(git describe HEAD)}
```

Comment:
Would it make sense to have a fallback here in case there's no value in e.g. shallow clones? Even a date stamp like `$(date +%s)` with some prefix, for example.

Speaking of shallow clones, it's worth checking that we're doing a full checkout on any GitHub Actions workflow that could run this.